PostGIS_Lidur3/Stadsetning.sql

```diff
@@ -0,0 +1,25 @@
+-- Finna allar staðsetningar sem tilheyra sjaldgæfustu staðsetningategundunum utan "The Seven Kingdoms"
+WITH type_counts AS (
+    -- Finna fjölda staðsetninga eftir staðsetningategund utan "The Seven Kingdoms"
+    SELECT l.type, COUNT(l.gid) AS location_count
+    FROM atlas.locations l
+    JOIN atlas.kingdoms k ON l.gid = k.gid
+    WHERE k.claimedby != 'The Seven Kingdoms'
+    GROUP BY l.type
+),
+min_count AS (
+    -- Finna minnsta fjölda staðsetninga fyrir staðsetningategundir
+    SELECT MIN(location_count) AS min_count
+    FROM type_counts
+)
+-- Sækja allar staðsetningar sem tilheyra tegundum með minnsta fjölda staðsetninga
+SELECT l.name, l.type
+FROM atlas.locations l
+JOIN atlas.kingdoms k ON l.gid = k.gid
```

Comment:
Þessir dálkar hafa ekkert að gera með hvorn annan. Þetta eru primary keys fyrir hvora töflu fyrir sig. 
```suggestion 
JOIN atlas.kingdoms k ON ST_Intersects(k.geog, l.geog)
```